beacon_node/lighthouse_network/src/rpc/codec/ssz_snappy.rs

```diff
@@ -472,7 +474,11 @@ fn handle_v1_request<T: EthSpec>(
         Protocol::Ping => Ok(Some(InboundRequest::Ping(Ping {
             data: u64::from_ssz_bytes(decoded_buffer)?,
         }))),
-
+        Protocol::LightClientBootstrap => Ok(Some(InboundRequest::LightClientBootstrap(
+            LightClientBootstrapRequest {
+                root: Hash256::from_ssz_bytes(decoded_buffer)?,
```

Comment:
I know the spec calls this `root`, but AFAIK the name doesn't really matter. We could call it `block_root` for clarity.